watchtower-cli/src/main/java/com/tracelink/appsec/watchtower/cli/reporter/AbstractReporter.java

```diff
@@ -0,0 +1,67 @@
+package com.tracelink.appsec.watchtower.cli.reporter;
+
+import com.tracelink.appsec.watchtower.cli.scan.UploadScanResult;
+import java.time.Duration;
+import java.util.Collections;
+import org.slf4j.Logger;
+
+/**
+ * Abstract class to report Watchtower scan results in various formats. Reports a summary of the
+ * scan results to the console.
+ *
+ * @author mcool
+ */
+public abstract class AbstractReporter {
+
+	private final Logger log;
```

Comment:
I think you can do Logger.getClass(this.getClass()) and it should get the concrete classname